apps/site-landing/src/containers/App/index.tsx

```diff
@@ -20,28 +18,29 @@ import { MediaContextProvider, mediaStyles } from '@thecointech/shared/component
 // Either import CSS or LESS;
 // - LESS is slower, but offers on-save hot-reload
 // - CSS is faster, but requires manual recompile
-import '../../semantic/semantic.css';
-//import '@thecointech/site-semantic-theme/semantic.less';
+//import '../../semantic/semantic.css';
+import '@thecointech/site-semantic-theme/semantic.less';
 import styles from './styles.module.less';
 
 export const App = () => {
   Prismic.useStore();
```

Comment:
Maybe this could be put in a context

Reply:
I'm not sure what you mean - like a react context?

Reply:
Yes. Putting the stores in a context and putting that context parent of everything is really a time saver.

Reply:
I think that's kind of what we already do - but we do it dynamically so nobody has to -know- ahead of time what's in the store.

We do that so we can split the store over shared/base/site(s) but each library can act as if it's in a sandbox.  It also means that we don't start a store if you don't use it.  This particular store probably should be moved to the "Help" page so if it isn't used it's not even started.  It's not a big deal for most things, but the account saga's are pretty heavy so it's mostly so we don't start them if we don't need them (ie, if you have a bunch of accounts - we only inject the store for the actual account we are using).

At least, that's what we are doing, which AFAIK (caveat - not much) is similar to what you are suggesting?

Reply:
(if there is a simple example of what you mean though, I'd love to check it out?  Might be a fancy new technique, and never know it could come in handy)